src/Auditable.php

```diff
@@ -282,18 +283,24 @@ public function toAudit(): array
 
         $tags = implode(',', $this->generateTags());
 
-        return $this->transformAudit([
```

Comment:
With just one `User` resolver, we would get the user at this point, like:

`$user = $this->resolveUser();`